common/handy/maps.go

```diff
@@ -0,0 +1,45 @@
+package handy
```

Comment:
Could you put these in utils or rename this as utils?

Reply:
There are too many utils I wanted to limit increase of that package.
I wasn't sure about adding the library for data structures. But more or less thats what it is. I will incorporate it.